security/kerberos_pac.ksy

```diff
@@ -0,0 +1,542 @@
+        type: filetime
+        doc: A FILETIME structure that contains the time at which the client's password expires.
+      - id: effective_name
+        -orig-id: EffectiveName
+        type: rpc_unicode_string
+        doc: An RPC_UNICODE_STRING structure that contains the user account's samAccountName attribute ([MS-ADA3] section 2.222) value.
+      - id: full_name
+        -orig-id: FullName
+        type: rpc_unicode_string
+        doc: An RPC_UNICODE_STRING structure that contains the user account's full name for interactive logon and is set to zero for network logon.
+      - id: logon_script
+        -orig-id: LogonScript
+        type: rpc_unicode_string
+        doc: An RPC_UNICODE_STRING structure that contains the user account's scriptPath attribute ([MS-ADA3] section 2.232) value for interactive logon
+      - id: profile_path
+        -orig-id: ProfilePath
+        type: rpc_unicode_string
+        doc: An RPC_UNICODE_STRING structure that contains the user account's profilePath attribute ([MS-ADA3] section 2.167) value for interactive logon
+      - id: home_directory
+        -orig-id: HomeDirectory
+        type: rpc_unicode_string
+        doc: An RPC_UNICODE_STRING structure that contains the user account's HomeDirectory attribute ([MS-ADA1] section 2.295) value for interactive logon
+      - id: home_directory_drive
+        -orig-id: HomeDirectoryDrive
+        type: rpc_unicode_string
+        doc: An RPC_UNICODE_STRING structure that contains the user account's HomeDrive attribute ([MS-ADA1] section 2.296) value for interactive logon
+      - id: logon_count
+        -orig-id: LogonCount
+        type: u2
+        doc: A 16-bit unsigned integer that contains the user account's LogonCount attribute ([MS-ADA1] section 2.375) value.
+      - id: bad_password_count
+        -orig-id: BadPasswordCount
+        type: u2
+        doc: A 16-bit unsigned integer that contains the user account's badPwdCount attribute ([MS-ADA1] section 2.83) value for interactive logon
+      - id: user_id
+        -orig-id: UserId
+        type: u4
+        doc: A 32-bit unsigned integer that contains the RID of the account
+      - id: primary_group_id
+        -orig-id: PrimaryGroupId
+        type: u4
+        doc: A 32-bit unsigned integer that contains the RID for the primary group to which this account belongs
+      - id: group_count
+        -orig-id: GroupCount
+        type: u4
+        doc: A 32-bit unsigned integer that contains the number of groups within the account domain to which the account belongs
+      - id: group_ids_referent
+        -orig-id: GroupIdsReferent
+        type: u4
+      - id: user_flags
+        -orig-id: UserFlags
+        type: u4
+        doc: A 32-bit unsigned integer that contains a set of bit flags that describe the user's logon information
+      - id: user_session_key
+        -orig-id: UserSessionKey
+        type: u1
+        repeat: expr
+        repeat-expr: 16
```

Comment:
This pattern

```ksy
  type: u1
  repeat: expr
  repeat-expr: {N}
```

should be almost always replaced with

```ksy
  size: {N}
```

It will be a byte array, no longer a list of `u1` integers, so it'll be probably easier to work with in the generated parser, too.